extensions/npm/package.json

```diff
@@ -289,6 +289,12 @@
           "tags": [
             "usesOnlineServices"
           ]
+        },
+        "npm.alwaysPerformLookup": {
+          "type": "boolean",
+          "description": "%config.npm.alwaysPerformLookup%",
+          "default": false,
+          "scope": "application"
```

Comment:
I am not entirely sure about the `scope` value here.

Reply:
`scope` should be `"resource"` so that it can be set in .vscode/settings.json